src/yea/ytest.py

```diff
@@ -311,8 +328,12 @@ def _run(self) -> None:
                     env[f"YEA_PLUGIN_{penv}_VALUES"] = json.dumps(pvalues)
             env["YEA_PLUGINS"] = ",".join(plugins)
 
+        start_time = time.monotonic()
```

Comment:
i hear people say it's best to use time.perf_counter for this purpose

Reply:
ill look into that in the future. thanks..